### DISCUSSION TITLE
Unable to do ammeter calibration

### DISCUSSION THREAD

**Discussion by Forhorse88** [OpenEVSE/open_evse#40] [Q&A]
Hello,
I have a difference between the current measurement made by openevse and the actual charging current.
My EVSE module arrived with firmware 8.2.2 I flashed it with firmware 8.2.3_EU but the problem is the same.
The RAPI command "$SA xxx y" reply "Error:RAPI_RESPONSE_BLOCKED"
The MQTT xxx/config/set "scale" have no effect
The HTTP API config reply "no modification" 
So the question is: how to proceed with the calibration of the current measurement?
Thanks in advance.

**Comment by kitizz** [comment]
I'm also seeing this issue

**Reply by lincomatic** [reply]
Post this as an issue to the WiFi firmware's repo. It blocks RAPI commands that it deems as "sensitive." I had to hack and build the f/w myself to get around it
